opm-core/Installation.rst

```diff
@@ -52,37 +57,34 @@ Then, using a superuser role::
     CREATE EXTENSION
 
 
-Then, you need to create a crontab that will process incoming data and dispatch them. As instance, to trigger it every minute::
+Then, you need to create a crontab that will process incoming data and dispatch them. For instance, to trigger it every minute::
 
     * * * * * psql -c 'SELECT wh_nagios.dispatch_record()' opm
 
 This crontab can belong to any user, as long as it can connect to the PostgreSQL
-opm database with any PostgreSQL role.
+``opm`` database with any PostgreSQL role.
 
-To import data in a warehouse, you need a PostgreSQL role. We recommand to
+To import data into a warehouse, you need a PostgreSQL role. We recommand to
 create a dedicated role, for instance::
 
-    postgres@opm=# CREATE USER user1 LOGIN PASSWORD 'password1';
+    postgres@opm=# CREATE ROLE opm LOGIN PASSWORD 'anothergoodpassword';
```

Comment:
Can you rename the new role to "opm_dispatcher", that'd be clearer.

Reply:
done